ReactWindows/ReactNative.Shared/UIManager/ReactShadowNode.cs

```diff
@@ -321,6 +325,15 @@ public YogaPositionType PositionType
         /// </summary>
         public virtual bool IsVirtualAnchor => false;
 
+        /// <summary>
+        /// Nodes that return <code>true</code> will be treated as relying on the child
+        /// for layout purposes.
+        /// </summary>
+        /// <remarks>
+        /// By default this method returns <code>false</code>.
+        /// </remarks>
+        public bool IsDelegatedLayout { get; }
```

Comment:
>IsDelegatedLayout [](start = 20, length = 17)

This seems to be an unrelated change - can we put this in a separate PR?

Reply:
It's actually used.

---
In reply to: [155706532](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r155706532) [](ancestors = 155706532)